src/gui/shareusergroupwidget.cpp

```diff
@@ -213,6 +228,8 @@ void ShareUserGroupWidget::slotSharesFetched(const QList<QSharedPointer<Share>>
     int height = 0;
     QList<QString> linkOwners({});
 
+    QPointer<ShareUserLine> justCreatedShareThatNeedsOptionalPassword = nullptr;
```

Comment:
Wow that's a mouthful :-D

I wonder if there's a slightly better name screaming to get out of this variable. Not sure which...

Also, are you sure you need a QPointer here? AFAICT there's no chance of going in the event loop and getting that object destroyed. A non owning raw pointer is very likely to suffice.

Reply:
@er-vin 

I like this comment :) Especially the first part :) Indeed, I wanted to make a **very** clear name :)

As for the QPointer, agree, well, that's just my panic fear of a crash.

Reply:
Well, then really better have a raw pointer. QPointer prevents moving away from QObject if needed while not bringing anything in that context.

Reply:
@er-vin Alrighty then :)